src/borgstore/backends/rclone.py

```diff
@@ -66,7 +67,7 @@ class Rclone(BackendBase):
     """
 
     precreate_dirs: bool = False
-    HOST = "localhost"
+    HOST = "127.0.0.1"
```

Comment:
I'll just note here that it is possible for `127.0.0.1` not to exist on IPv6 only computers... `localhost` should map to `::0` in that case. But then again there are probably computers where `localhost` doesn't work too!

Reply:
I also wondered about the localhost change. @alexandru-bagu was there a specific reason why you changed that?

Reply:
Don't remember the exact reason right now but I remember that when I was checking if the port was bound, the check would fail even though rclone was listening to the specified port.
It might be a Cygwin issue only and an option could be to allow the user to specify the hostname for binding the socket.

Reply:
ok, so guess we'll leave that "as is".